components/profile/tabs/information.js

```diff
@@ -0,0 +1,279 @@
+import React, { useEffect, useState } from 'react'
+import { 
+    Button,
+    Container,
+    Dialog,
+    DialogTitle,
+    DialogContent,
+    DialogActions,
+    Grid,
+    Paper,
+    MenuItem,
+    TextField,
+    Select,
+    Skeleton,
+    styled,
+} from '@mui/material';
+import { Icon } from '@iconify/react';
+import axiosClient from '../../../axiosClient';
+
+const genderValues = ['Male', 'Female', 'Unknown'];
+const userTypeValues = ['Basic', 'Moderator', 'Adminstrator'];
+
+const Label = styled('div')(({ theme }) => ({
+    ...theme.typography.h6,
+    padding: theme.spacing(1),
+    textAlign: 'left',
+    color: theme.palette.text.primary,
+}));
+
+const Value = styled('div')(({ theme }) => ({
+    ...theme.typography.body2,
+    padding: theme.spacing(1),
+    textAlign: 'left',
+    color: theme.palette.text.secondary,
+}));
+
+const Config = styled('div')(({ theme }) => ({
+    ...theme.typography.body2,
+    padding: theme.spacing(1),
+    position: 'relative',
+    color: theme.palette.text.primary,
+}));
+
+const ConfigButton = styled(Button)(({ theme }) => ({
+    position: 'absolute',
+    right: '15px',
+}));
+
+const EditTextInput = (props) => {
+    const {setCallbackValue, defaultValue} = props;
+    const handleChange = (event) => {
+        setCallbackValue(event.target.value);
+    }
+    return (
+        <TextField
+                autoFocus
+                margin="dense"
+                id="edit-information-text-input"
+                type="text"
+                defaultValue={defaultValue}
+                fullWidth
+                variant="standard"
+                onChange={handleChange}
+            />
+    );
+}

```

Comment:
- I see this is a component. Could you extract this to a seperate file. same with `EditSelect` `EditDialog` `ProfileCell`